### PR TITLE
Sixpack docs switcharoo

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?
- Adds the sixpack AB testing doc from the wiki over to our gitbooks docs folder
- Expands a bit on the instructions related to the `experiments.json` schema
- Adds Traffic Fraction data to those instructions.

### What are the relevant tickets/cards?
#631

### CODE REVIEW

File: docs/development/Sixpack AB Testing/sixpack-ab-testing.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+
+# Implementing Sixpack on the front-end
+
+When implementing an A/B test using Sixpack in Phoenix-Next on the front-end, there are a few steps for successfully executing an experiment.
+
+## Step 1
+First, you need to edit the experiments JSON file located in the `/resources/assets` directory, and designate the relavant data for the experiment.
+The hierarchy is as follows:
```
Comment (mendelB) [inline docs/development/Sixpack AB Testing/sixpack-ab-testing.md]: This is the only portion that I've updated for now, as compared to [the wiki](https://github.com/DoSomething/phoenix-next/wiki/Sixpack-AB-Testing):
## Step 1
First, you need to edit the experiments JSON file located in the `/resources/assets` directory, and designate the experiment name, all alternatives, and any additional meta data, such as whether a condition must be met before an experiment can execute:

```javascript
// Example experiments.json

{
  "lede_banner_number_of_buttons": {
    "meta": {
      "condition": "unaffiliated"
    },
    "alternatives": {
      "a": "one_button",
      "b": "two_buttons"
    }
  }
}
```

Might be worth revisiting and updating this whole doc more thoroughly.

**Comment by mendelB** [comment]
I removed the Sixpack AB Testing directory, and just added it in as a file, as I think that made more sense. Also updated the `SUMMARY.md`.

File: docs/development/sixpack-ab-testing.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+
+# Implementing Sixpack on the front-end
```
Comment (weerd) [inline docs/development/sixpack-ab-testing.md]: Remove the empty space above and add a horizontal rule `***` under this page title to follow format of other pages :)

Comment (weerd) [reply docs/development/sixpack-ab-testing.md]: Also capitalize first letters on all words in H1 titles :)

At least it's the standard I'm trying to force 😬

Comment (mendelB) [reply docs/development/sixpack-ab-testing.md]: Done and done :)

**Pull request merged**